File: app/src/main/java/com/example/maksim/testapp/details/model/repository/LocalDetailsRepositoryListener.java
package com.example.maksim.testapp.details.model.repository;

import com.example.maksim.testapp.details.model.data.GitHubUserDetails;

/**
 * Created by Maksim on 2017-09-10.
 */

public interface LocalDetailsRepositoryListener {
    void onUserDetailsLoaded(GitHubUserDetails userDetails);
}

File: app/src/main/java/com/example/maksim/testapp/details/model/repository/LocalDetailsRepositoryListener.kt
package com.example.maksim.testapp.details.model.repository

import com.example.maksim.testapp.details.model.data.GitHubUserDetails

/**
 * Created by Maksim on 2017-09-10.
 */

interface LocalDetailsRepositoryListener {
    fun onUserDetailsLoaded(userDetails: GitHubUserDetails?)
}
